src/assessment/builder.py

```diff
@@ -50,11 +50,13 @@
                 </html>
                """
 
+CONFIRMATION_PROMPT = "\nDo you need to modify any of the values for this campaign?"
+
 
 def set_time_zone():
     """Select a timezone from a list of US-based time zones.
 
-    :return: Time zone name based on pytz.
+    : return Time zone name based on pytz.
     """
     # TODO Allow for a select more option to get access to full list of Time Zones
```

Comment:
There should be a corresponding GitHub issue created and issue URL added to this TODO comment.

Reply:
Good Catch @dav3r! The TODO was already there and I didn't realize there wasn't an issue for it. I'll get it added in. Thanks!

Reply:
Addressed in ece7e9a.

Reply:
Added link to issue created here: 171b505dece98ef18071edd28c95e49a2984c02c